.github/workflows/pr-summary.yml

```
@@ -0,0 +1,21 @@
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  summarize:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: ashgw/pr-summarizer
```

actionlint (1.7.4)

specifying action "ashgw/pr-summarizer" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)